### ISSUE TITLE
Bug: Native browser of the plugin does not update when presets are skipped in the DAW

### ISSUE THREAD

**Issue by MrPlugDev** [JeffMcClintock/SynthEdit-Tracking#47]
Description:
When presets are skipped or changed within the DAW, the native browser of the plugin does not update to reflect these changes, and vice versa. This mismatch results in a disjointed user experience and makes it difficult to manage presets effectively. Synchronization between the DAW's preset selection and the plugin's native browser is needed to maintain consistency and improve usability.

Steps to reproduce:

-Load the plugin in a DAW and select a preset.
-Skip or change presets within the DAW.
-Observe that the native browser of the plugin does not update to reflect the changes made in the DAW.
-Similarly, changing presets in the plugin’s native browser does not update the DAW's preset selection. (some exceptions).

Additional Information:
This lack of synchronization affects the user experience and presents a basic need for improved integration between the DAW and the plugin's native browser. Notably, all plugins from other manufacturers handle this synchronization correctly, highlighting the need for this functionality in SynthEdit plugins.

Tag: #Bug

**Comment by JeffMcClintock** [comment]
@MrPlugDev 

I tested this in Cubase 10.5 with a basic plugin.
When I changed the preset from Cubases 'Load Preset' menu, the plugins preset browser switched to the correct preset.

It might be that a preset contains one or more parameters belonging to Meters, in which case two copies of the same preset might actually contain different values.

How to check for this:
Load the preset from the plugins own UI. Save the preset somewhere in XML format.
Load the same preset from the DAW. Save this preset also in XML format.

Compare the two files, look for any differences that might cause the plugin to treat the two presets as different.
